frontend/packages/console-shared/src/utils/sample-utils.ts

```diff
@@ -348,6 +348,32 @@ const defaultSamples = (t: TFunction) =>
           },
           targetResource: getTargetResource(ConsoleOperatorConfigModel),
         },
+        {
+          title: t('console-shared~Add user perspectives'),
+          description: t(
+            'console-shared~Provides a list of all the available user perspectives which are shown in the perspective dropdown. The perspectives must be added below spec customization.',
+          ),
+          id: 'user-perspectives',
+          snippet: true,
+          lazyYaml: () => {
+            return new Promise<string>((resolve) => {
+              const unsubscribe = subscribeToExtensions<Perspective>(
+                (extensions: LoadedExtension<Perspective>[]) => {
+                  const yaml = extensions
+                    .map((extension) => {
+                      const { id } = extension.properties;
+                      return ` - id: ${id}\n   visiblity:\n    state: Enabled\n`;
```

Comment:
`visiblity` => `visibility `

You can also return an object here. The snippet direct above only used string concatenation to add some comments to the output. This is useful if the ID is not clear enough. The comment is only visible with Show YAML in the sidebar. When its inserted it was unfortunately not added.

```suggestion
                      return {
                        id,
                        visibility: {
                          state: 'Enabled',
                        },
                      };
```